core/src/main/java/fr/sncf/osrd/railjson/schema/schedule/RJSAllowance.java

```diff
@@ -18,14 +18,16 @@
         public double endPosition = Double.NaN;
         @Json(name = "capacity_speed_limit")
         public double capacitySpeedLimit = -1;
+        @Json(name = "value")
```

Comment:
```suggestion
```

This is only needed when the name is different (generally to go from camelCase to snake_case)